ostree-simplified-installer.sh

```diff
@@ -1385,6 +1394,11 @@ sudo composer-cli compose delete "${COMPOSE_ID}" > /dev/null
 sudo composer-cli blueprints delete upgrade > /dev/null
 
 greenprint "🗳 Upgrade ostree image/commit"
+# Update default Fedora's repository
+if [[ "$ID" == fedora ]]; then
+    sudo ssh "${SSH_OPTIONS[@]}" -i "${SSH_KEY}" ${BLUEPRINT_USER}@${ROOT_CERT_GUEST_ADDRESS} "echo ${EDGE_USER_PASSWORD} |sudo -S ostree remote delete ${REF_PREFIX}"
+    sudo ssh "${SSH_OPTIONS[@]}" -i "${SSH_KEY}" ${BLUEPRINT_USER}@${ROOT_CERT_GUEST_ADDRESS} "echo ${EDGE_USER_PASSWORD} |sudo -S ostree remote add --no-gpg-verify ${REF_PREFIX} https://ostree.fedoraproject.org/iot"
```

Comment:
It's not `https://ostree.fedoraproject.org/iot`, but `${PROD_REPO_URL}`.

Reply:
Thanks! Done.